CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Added
+* **dso-toolkit** Vaste toelichting als variant bij info.njk ([#907](https://github.com/dso-toolkit/dso-toolkit/issues/907))
```

Comment:
Ik vind dit een beetje een rare beschrijving. Beter is denk ik: 
- "Info-component zonder sluitmogelijkheid"
Je gaat toch niks over de achterliggende 'techniek' in zo'n beschrijving opnemen `info.njk` ?